Cypress/cypress/integration/Parallel_Tests/Create_Workflow/customWorkflow.spec.js

```diff
@@ -145,6 +158,12 @@ describe("Testing the validation of the final verdict without target application
 
 	it("Validating graph nodes", () => {
 		cy.validateWorkflowStatus(workflowName, workflowNamespace, ["Running", "Failed"]);
+		cy.get("table")
+			.find("tr")
+			.eq(1)
+			.find('td')
+			.eq(0)
+			.click({ scrollBehavior: false });
```

Comment:
I think we were searching for workflow name before clicking on it? Can we have that back?